winpr/libwinpr/sspi/Schannel/schannel_openssl.c

```diff
@@ -237,7 +235,7 @@ SECURITY_STATUS schannel_openssl_client_process_tokens(SCHANNEL_OPENSSL* context
 			if (!pBuffer)
 				return SEC_E_INVALID_TOKEN;
 
-			status = BIO_write(context->bioRead, pBuffer->pvBuffer, pBuffer->cbBuffer);
+			BIO_write(context->bioRead, pBuffer->pvBuffer, pBuffer->cbBuffer);
```

Comment:
just an idea: if this code is WIP, perhaps we should remove it from master and you would keep it on a private branch until it is stabilized. That way we keep master with less warnings, and we don't fool people to think schannel_openssl is functional. Just an idea !